packages/apps/frontera/src/routes/finder/src/components/Columns/contacts/Cells/organization/OrganizationNameCell.tsx

```diff
@@ -38,6 +38,14 @@ export const OrganizationNameCell = observer(
 
     const lastPositionParams = tabs[orgId];
     const href = getHref(orgId, lastPositionParams);
+    const contactStore = store.contacts.value.get(contactId);
+
+    const enrichedContact = contactStore?.value.enrichDetails;
+
+    const enrichingStatus =
+      !enrichedContact?.enrichedAt &&
+      enrichedContact?.requestedAt &&
+      !enrichedContact?.failedAt;
```

Comment:
those can probably be turned into a store getter for convenience

Reply:
As discussed in the call it's beneficial to leave it as better alternative to showing "unnamed" and then only when BE sends update showing enriching copy